unison-src/builtin-tests/base.md

```diff
@@ -5,7 +5,7 @@ Thus, make sure the contents of this file define the contents of the cache
 (e.g. don't pull `latest`.)
 
 ```ucm
-.> pull @unison/base/releases/2.5.0 .base
-.> builtins.mergeio
-.> undo
+scratch/main> pull @unison/base/releases/2.5.0 .base
+scratch/main> builtins.mergeio
```

Comment:
might `lib.install` here and `builtins.mergeio lib.builtins` but it can be some other time

Reply:
I'm going to leave that for now since it's not strictly necessary and this diff is already hugemongous :)